crates/stc_ts_type_checker/tests/conformance/override/override15.stats.rust-debug

```diff
@@ -1,6 +1,6 @@
 Stats {
     required_error: 1,
     matched_error: 0,
-    extra_error: 0,
+    extra_error: 1,
```

Comment:
It seems like the error is divided into TS4113 and TS4117 whether or not spell fixes can be performed :(

TS4112: https://github.com/microsoft/TypeScript/blob/c61abc9d830c3bc767b42a50e7b8adbba4bf13f0/src/compiler/diagnosticMessages.json#L4011-L4014
TS4117: https://github.com/microsoft/TypeScript/blob/c61abc9d830c3bc767b42a50e7b8adbba4bf13f0/src/compiler/diagnosticMessages.json#L4031-L4034

Reply:
Let's normalize them for now.

https://github.com/dudykr/stc/blob/83931f135bbf23fa4bef72437f9487ceb28f7572/crates/stc_ts_errors/src/lib.rs#L1601-L1679